cookbooks/swift/templates/default/etc/swift/proxy-server/default.conf-template.erb

```diff
@@ -49,6 +49,7 @@ s3_acl = True
 check_bucket_owner = True
 force_swift_request_proxy_log = true
 ratelimit_as_client_error = true
+s3_inventory_enabled = true
```

Comment:
not strictly related to testing with containers - but super useful when testing ss-release-* tags w/ nv-swift-inventory